server/routes/user.js

```diff
@@ -31,6 +31,10 @@ router.route('/:userId/device/:deviceId')
 
   .delete(validate(paramValidation.deviceById), userCtrl.removeDevice);
 
+router.route('/:userId/group')
```

Comment:
I agree that it works in the singular for our other APIs, but those are actually only ever getting or updating one document, where this is often getting many.